src/sd_jwt/bin/generate.py

```diff
@@ -46,6 +46,7 @@ def generate_test_case_data(settings: Dict, testcase_path: Path, type: str):
     }
 
     claims.update(testcase["user_claims"])
+    claims = {key: value for key, value in claims.items() if value is not None}
```

Comment:
This removes any `null`-valued claims, but they might be used in some contexts. I suggest to change lines 42-46 to something like the following:
```
claims = {}

if not "iss" in testcase["user_claims"] or testcase["user_claims"] is None:
    claims["iss"] = settings["identifiers"]["issuer"]
 ```

Reply:
what about not having any user claims?

````
_uc = testcase.get("user_claims", {})
if not _uc or "iss" not in _uc:
````

considering that just `if iss not in _uc` would be enough since if _uc is {} ...

Reply:
No user claims at all is not a case we need to care about IMO.

Reply:
As we discussed briefly, I"m going to add an include_default_claims flag that itself defaults to true to the specification.yml rather than this null value thing.